packages/js/src/account/account.ts

```diff
@@ -188,6 +187,9 @@ export class Account implements NearAccount {
       isSubAccount?: boolean;
     } = {},
   ): Promise<NearAccount> {
+    const randomNumber = Math.floor((Math.random() * (9999 - 1000)) + 10_000);
+    const accountId = `dev-${randomNumber}.${this.accountId}`;
```

Comment:
It feels like we should make this consistent between all tooling otherwise people might confuse about why they're different between the RS and JS versions. RS has it the same as CLI, so I feel we should lean into that format of `dev-{randomNumber}-{DateTime::now}`